design/object-bucket.md

```diff
@@ -0,0 +1,167 @@
+# Rook Object Store Bucket
```

Comment:
Beyond the design doc (intended audience is developers), we need documentation for admins. Either we could add a new section to the ceph-object.md page, or a new page just below this in the TOC that discusses creating buckets. The yamls mentioned in the ceph-examples.md are a start, but the user really needs more info. Some of this design doc could even be moved/copied to the documentation, but not with all the design details.

Reply:
+1

Reply:
See #3655